sql/schemas/bookbrainz.sql

```diff
@@ -833,57 +836,71 @@ CREATE VIEW bookbrainz.edition AS
+		e.bbid, wd.id AS data_id, wr.id AS revision_id, (wr.id = w.master_revision_id) AS master, wd.annotation_id, dis.comment,
+		als.default_alias_id, al."name", al.sort_name, wd.type_id, worktype.label as work_type, wd.alias_set_id, wd.identifier_set_id,
 		wd.relationship_set_id, e.type, wd.language_set_id
 	FROM bookbrainz.work_revision wr
 	LEFT JOIN bookbrainz.entity e ON e.bbid = wr.bbid
 	LEFT JOIN bookbrainz.work_header w ON w.bbid = e.bbid
 	LEFT JOIN bookbrainz.work_data wd ON wr.data_id = wd.id
 	LEFT JOIN bookbrainz.alias_set als ON wd.alias_set_id = als.id
+	LEFT JOIN bookbrainz.alias al ON al.id = als.default_alias_id
+	LEFT JOIN bookbrainz.disambiguation dis ON dis.id = wd.disambiguation_id
+	LEFT JOIN bookbrainz.publisher_type worktype ON worktype.id = wd.type_id
 	WHERE e.type = 'Work';
 
 CREATE VIEW bookbrainz.publisher AS
 	SELECT
-		e.bbid, psd.id AS data_id, psr.id AS revision_id, (psr.id = ps.master_revision_id) AS master, psd.annotation_id, psd.disambiguation_id,
-		als.default_alias_id, psd.begin_year, psd.begin_month, psd.begin_day,
+		e.bbid, psd.id AS data_id, psr.id AS revision_id, (psr.id = ps.master_revision_id) AS master, psd.annotation_id, dis.comment,
+		als.default_alias_id, al."name", al.sort_name, psd.begin_year, psd.begin_month, psd.begin_day,
 		psd.end_year, psd.end_month, psd.end_day, psd.ended, psd.area_id,
-		psd.type_id, psd.alias_set_id, psd.identifier_set_id, psd.relationship_set_id, e.type
+		psd.type_id, pubtype.label as publisher_type, psd.alias_set_id, psd.identifier_set_id, psd.relationship_set_id, e.type
 	FROM bookbrainz.publisher_revision psr
 	LEFT JOIN bookbrainz.entity e ON e.bbid = psr.bbid
 	LEFT JOIN bookbrainz.publisher_header ps ON ps.bbid = e.bbid
 	LEFT JOIN bookbrainz.publisher_data psd ON psr.data_id = psd.id
 	LEFT JOIN bookbrainz.alias_set als ON psd.alias_set_id = als.id
+	LEFT JOIN bookbrainz.alias al ON al.id = als.default_alias_id
+	LEFT JOIN bookbrainz.disambiguation dis ON dis.id = psd.disambiguation_id
+	LEFT JOIN bookbrainz.publisher_type pubtype ON pubtype.id = psd.type_id
 	WHERE e.type = 'Publisher';
 
 CREATE VIEW bookbrainz.edition_group AS
 	SELECT
-		e.bbid, pcd.id AS data_id, pcr.id AS revision_id, (pcr.id = pc.master_revision_id) AS master, pcd.annotation_id, pcd.disambiguation_id,
-		als.default_alias_id, pcd.type_id, pcd.author_credit_id, pcd.alias_set_id, pcd.identifier_set_id,
+		e.bbid, pcd.id AS data_id, pcr.id AS revision_id, (pcr.id = pc.master_revision_id) AS master, pcd.annotation_id, dis.comment,
+		als.default_alias_id, al."name", al.sort_name, pcd.type_id, egtype.label as edition_group_type, pcd.author_credit_id, pcd.alias_set_id, pcd.identifier_set_id,
 		pcd.relationship_set_id, e.type
 	FROM bookbrainz.edition_group_revision pcr
 	LEFT JOIN bookbrainz.entity e ON e.bbid = pcr.bbid
 	LEFT JOIN bookbrainz.edition_group_header pc ON pc.bbid = e.bbid
 	LEFT JOIN bookbrainz.edition_group_data pcd ON pcr.data_id = pcd.id
```

Comment:
maybe change the alias of these so that it makes sense compared to the table name